packages/block-library/src/block/edit.js

```diff
@@ -159,6 +162,7 @@ export default compose( [
 			isFetching: isFetchingReusableBlock( ref ),
 			isSaving: isSavingReusableBlock( ref ),
 			block: reusableBlock ? getBlock( reusableBlock.clientId ) : null,
+			canUpdateBlock: !! reusableBlock && ! reusableBlock.isTemporary && canUser( 'update', 'blocks', ref ),
```

Comment:
> Any reason you're checking `!! reusableBlock` rather than just using `reusableBlock`? Just curious!

If a value is falsey in this scenario, the specific value is returned in the left-hand of the condition.

If `reusableBlock` was `undefined` for example, without the `!!`, `canUpdateBlock` would be passed as `undefined`, despite its name indicating that it would be a boolean. As written, it satisfies an implicit contract of providing a boolean value. Practically speaking this would rarely have much an impact, except in a scenario where someone expected it to be the particular boolean form (`=== false`).